muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/organizationmanagement/rest/OrganizationUserManagementRESTService.java

```diff
@@ -171,6 +172,69 @@ public Response searchStaffMembers(
     return Response.ok(responseStaffMembers).build();
   }
   
+  @GET
+  @Path("/studentsSummary")
+  @RESTPermit(OrganizationManagementPermissions.ORGANIZATION_VIEW)
+  public Response studentsSummary(
+      @QueryParam("firstResult") @DefaultValue("0") Integer firstResult,
```

Comment:
Since this endpoint returns the total number of active/inactive students, it shouldn't require the frontend to specify a search range...